<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="1" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>160</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>分时</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="KMapWidget" name="kmap" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>日K</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbDate">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>20/01/01/三</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbOpen">
        <property name="text">
         <string>开</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbClose">
        <property name="text">
         <string>收</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbHigh">
        <property name="text">
         <string>高</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbLow">
        <property name="text">
         <string>低</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbVolume">
        <property name="text">
         <string>量</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbAmount">
        <property name="text">
         <string>额</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbChg">
        <property name="text">
         <string>涨 </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QListWidget" name="listWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_A">
    <property name="title">
     <string>菜单A</string>
    </property>
    <addaction name="actionmenu1"/>
    <addaction name="actionmenu2"/>
   </widget>
   <widget class="QMenu" name="menu_B">
    <property name="title">
     <string>菜单B</string>
    </property>
   </widget>
   <addaction name="menu_A"/>
   <addaction name="menu_B"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="actionAdd2"/>
  </widget>
  <action name="actionmenu1">
   <property name="text">
    <string>刷新</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionmenu2">
   <property name="text">
    <string>menu2</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="qtdemo.qrc">
     <normaloff>:/toolbar/Res/360AntiVirus_32.png</normaloff>
     <normalon>:/toolbar/D:/BaiduNetdiskDownload/QT/Qt360-10.0/resource/advtools/360AntiVirus_32.png</normalon>:/toolbar/Res/360AntiVirus_32.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionAdd2">
   <property name="icon">
    <iconset resource="qtdemo.qrc">
     <normaloff>:/toolbar/Res/360EntAdmin_32.png</normaloff>
     <normalon>:/toolbar/D:/BaiduNetdiskDownload/QT/Qt360-10.0/resource/advtools/360ask_32.png</normalon>:/toolbar/Res/360EntAdmin_32.png</iconset>
   </property>
   <property name="text">
    <string>Add2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KMapWidget</class>
   <extends>QWidget</extends>
   <header>KMapWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qtdemo.qrc"/>
 </resources>
 <connections/>
</ui>
